PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -123,6 +123,11 @@ namespace Poly
 		/// <param name="pos">New wheel position.</param>
 		void UpdateWheelPos(const Vector& pos) { InputEventsQueue.PushBack({eInputEventType::WHEELMOVE, pos}); }
 
+		//functions for closing the game
+		bool ShouldQuit();
```

Comment:
`ShouldQuit` should be implemented as const method (`void ShouldQuit() const;`). Also please provide XML documentation for this method describing how is it used. I also suggest rename to `IsGameQuiteRequested`.